lambdas/jwtverification/src/test/java/uk/gov/di/ipv/cri/passport/jwtverification/JwtVerificationHandlerTest.java

```diff
@@ -105,4 +139,41 @@ void shouldReturn400IfFailedToParseJWT() throws JsonProcessingException {
         assertEquals(ErrorResponse.FAILED_TO_PARSE_SHARED_ATTRIBUTES_JWT.getCode(), error.get("code"));
         assertEquals(ErrorResponse.FAILED_TO_PARSE_SHARED_ATTRIBUTES_JWT.getMessage(), error.get("message"));
     }
+
+    @Test
+    void shouldReturn400IFClientIdIsNotSet() throws JsonProcessingException {
+        var event = new APIGatewayProxyRequestEvent();
+
+        var response = underTest.handleRequest(event, context);
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        Map<String, Object> error = objectMapper.readValue(response.getBody(), new TypeReference<>() {});
+        assertEquals(400, response.getStatusCode());
+        assertEquals(ErrorResponse.MISSING_CLIENT_QUERY_PARAMETERS.getCode(), error.get("code"));
+        assertEquals(ErrorResponse.MISSING_CLIENT_QUERY_PARAMETERS.getMessage(), error.get("message"));
+    }
+
+    @Test
+    void shouldReturn200WithValidCertificate() throws CertificateException {
```

Comment:
I don't think this test is actually testing anything valuable here. At the moment its just asserting that your mocked configurationService returns a x509 certificate. Its not actually testing anything from our actual code base I think?

I think maybe it can just be removed?

Reply:
Removed test